src/include/miopen/fusion/utils.hpp

```diff
@@ -81,8 +81,6 @@ inline bool WinoCommonIsApplicable(const FusionContext& params)
 
     if(!conv_ctx.problem.Is2d())
         return false;
-    if(!conv_ctx.problem.IsFp32())
```

Comment:
@Slimakanzer a few lines below, do we know if XNACK settings could be enabled?
```
if(target.Xnack() && *target.Xnack())
```

Reply:
@junliume IIRC Winograd kernels work correctly only if XNACK is not enabled on target. Perhaps this is possible to create Winograd kernels that support XNACK-enabled targets.

@Kirpich30000 Is it possible, what do you think?